### DISCUSSION TITLE
how can i install code server on raspberry pi?

### DISCUSSION THREAD

**Discussion by sebasop320** [coder/code-server#4078] [Q&A] [answered]
Hello, i need some help instaling code server in my raspberry pi 4B 

pls help me.

Thanks!

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
All you need is NodeJS, cURL, and some basic dependencies. Run the install script on the Readme, and you are good to go.

**Reply by jsjoeio** [reply]
Take a look at the docs [here](https://coder.com/docs/code-server/v3.11.1/install#raspberry-pi) and let us know if you get stuck.

**Reply by sebasop320** [reply]
It did not work. But i am goin to install a other operating system on mi Pi dose ubuntu server line work?

**Reply by jsjoeio** [reply]
> It did not work.

Can you share the logs?

> But i am goin to install a other operating system on mi Pi dose ubuntu server line work?

Ubuntu should work just fine. Check the docs [here](https://coder.com/docs/code-server/v3.11.1/install#debian-ubuntu)

**Reply by lg-nineoneninethreenine** [reply]
Ubuntu Server will be fine. After installing, make sure you have the basic dependencies, and then run the cURL line in the readme.
